### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by tewchen**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -312,3 +312,22 @@ DeviceEvents
...
+## Frequently asked questions
+**What is the removable storage media limitation for the maximum number of USBs?**
+
+- Internally we have validated one USB group with 100,000 media - up to 7 MB in size. The policy works fine in both Intune and GPO without performance issues.
+
+**Why does the policy not work?**
+
+- The most common reason is there is no required [Anti-malware Client Version](/microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control?view=o365-worldwide#prepare-your-endpoints).
+
+- Or the XML file is not correctly formatted, for example, forget to replace & with &amp; in the XML file; or the text editor might add a byte order mark (BOM) 0xEF 0xBB 0xBF at the beginning of the files which causes the XML parsing not to work. One simple way is to download  [sample file](https://github.com/microsoft/mdatp-devicecontrol/tree/main/Removable%20Storage%20Access%20Control%20Samples) file (click 'Raw' and then 'Save as') and then update.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
- Or the XML file is not correctly formatted, for example, not using the correct markdown formatting for the "&" character in the XML file, or the text editor might add a byte order mark (BOM) 0xEF 0xBB 0xBF at the beginning of the files which causes the XML parsing not to work. One simple solution is to download the [sample file](https://github.com/microsoft/mdatp-devicecontrol/tree/main/Removable%20Storage%20Access%20Control%20Samples) (select **Raw** and then **Save as**) and then update.
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 2]
```diff
@@ -312,3 +312,22 @@ DeviceEvents
...
+
+- If the value is there and if the policy is managed via Group Policy, check whether the client machine can access the policy XML path.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
- If there is a value and the policy is managed via Group Policy, check whether the client device can access the policy XML path.
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 3]
```diff
@@ -312,3 +312,22 @@ DeviceEvents
...
+
+
+
+
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tewchen, Kindly commit my suggested changes if you agree with them. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @tewchen @yogkumgit @JohanFreelancer9

**Pull request merged**